tornado/log.py

```diff
@@ -146,14 +150,12 @@ def __init__(self, fmt=DEFAULT_FORMAT, datefmt=DEFAULT_DATE_FORMAT,
                 for levelno, code in colors.items():
                     self._colors[levelno] = unicode_type(curses.tparm(fg_color, code), "ascii")
                 self._normal = unicode_type(curses.tigetstr("sgr0"), "ascii")
-            elif sys.stderr is getattr(colorama, 'wrapped_stderr', object()):
+            elif sys.stderr is getattr(colorama.initialise, 'wrapped_stderr', object()):
```

Comment:
`colorama.initialize` will raise if `colorama` is None (i.e. if neither curses or colorama is present. We should really test both with and without colorama (we do similar tests on travis), but appveyor is really slow and makes that painful). 

I think spreading the logic for colorama between this line and `_stderr_supports_color` is error-prone. Why does `_stderr_supports_color` look at `wrapped_stderr` but not `sys.stderr`? I think a better fix is to move this check into `_stderr_supports_color`, and then we can use hard-coded ANSI color codes (i.e. the next three lines) whenever that function returns true and `curses` is not present.